lib/ansible/modules/cloud/docker/docker_compose.py

```diff
@@ -1037,8 +1050,11 @@ def cmd_scale(self):
                     result['changed'] = True
                     service_res['scale'] = scale - len(containers)
                     if not self.check_mode:
+                        out_redir_name, err_redir_name = make_redirection_tempfiles()
                         try:
-                            service.scale(scale)
+                            with stdout_redirector(out_redir_name):
+                                with stderr_redirector(err_redir_name):
+                                    service.scale(scale)
                         except Exception as exc:
                             self.client.fail("Error scaling %s - %s" % (service.name, str(exc)))
```

Comment:
```suggestion
                            fail_reason = get_failure_info(exc, out_redir_name, err_redir_name,
                                                           msg_format="Error scaling {0} - %s".format(service.name))
                            self.client.fail(**fail_reason)
                        else:
                            cleanup_redirection_tempfiles(out_redir_name, err_redir_name)
```

Reply:
Not sure why github isn't saying that this is outdated... but this is fixed.